af/analysis/rotation.py

```diff
@@ -201,6 +241,136 @@ def drain_droplet_if_needed(db_conn, live_droplets, active_droplets_count):
         drain_droplet_in_db_table(db_conn, oldest)
 
 
+def create_le_do_ssl_cert():
+    """Create Let's Encrypt SSL Certificate through the Digital Ocean API
+
+    Namecheap DNS entry:
+        th.ooni.org
+            NS  ns1.digitalocean.com
+            NS  ns2.digitalocean.com
+            NS  ns3.digitalocean.com
+
+    """
+    # debdeps: certbot python3-certbot-dns-digitalocean
+
+    cmd = [
+        "certbot",
+        "certonly",
+        "--dns-digitalocean",
+        "--dns-digitalocean-credentials",
+        certbot_creds,
+        "-d",
+        "'*.th.ooni.org'",
+    ]
```

Comment:
Shouldn't this command be run in here somehow?

Reply:
It's commented out and there's a TODO for it